sklearn/preprocessing/tests/test_common.py

```diff
@@ -45,8 +52,19 @@ def _get_valid_samples_by_column(X, col):
         (RobustScaler(with_centering=False), robust_scale, True, False, []),
     ],
 )
+@pytest.mark.parametrize(
+    "sparse_container",
+    [
+        CSR_CONTAINERS,
+        CSC_CONTAINERS,
+        COO_CONTAINERS,
+        LIL_CONTAINERS,
+        DOK_CONTAINERS,
+        BSR_CONTAINERS,
+    ],
```

Comment:
Done this way, chances are you might be iterating through a list-of-lists instead of a single list at a time as you probably intended